store/mockstore/mocktikv/executor.go

```diff
@@ -72,12 +73,17 @@ func (e *tableScanExec) GetSrcExec() executor {
 	return e.src
 }
 
-func (e *tableScanExec) ResetCount() {
-	e.count = 0
+func (e *tableScanExec) ResetCounts() {
+	e.counts[e.start] = 0
 }
 
-func (e *tableScanExec) Count() int64 {
-	return e.count
+func (e *tableScanExec) Counts() []int64 {
```

Comment:
The function name looks like its read-only, but it also updates the `start` and it's not repeatable.

Reply:
Any suggestion?